testing/README.md

```diff
@@ -87,6 +88,22 @@ Deno.test("isNotStrictlyEqual", function (): void {
 });
 ```
 
+Using `assertSnapshot()`:
+
+For more usage information, see [Snapshot Testing](#snapshot-testing).
+
+```ts
+import { assertSnapshot } from "https://deno.land/std@$STD_VERSION/testing/snapshot.ts";
+
+Deno.test("isSnapshotMatch", function (test): void {
```

Comment:
nit: I think most docs/tests use `t` for TestContext variable. Let's make it consistent

```suggestion
Deno.test("isSnapshotMatch", function (t): void {
```